sdk/face/azure-ai-vision-face/src/main/java/com/azure/ai/vision/face/ClientUtils.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.vision.face;
+
+import com.azure.ai.vision.face.models.FaceAttributeType;
+import com.azure.ai.vision.face.models.FaceDetectionModel;
+import com.azure.ai.vision.face.models.FaceRecognitionModel;
+import com.azure.core.http.rest.RequestOptions;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+class ClientUtils {
```

Comment:
Can we move this to implementation package?

Reply:
Done.